master/internal/trial.go

```diff
@@ -439,14 +437,13 @@ func (t *trial) runningReceive(ctx *actor.Context) error {
 			return err
 		}
 
-	case actor.ChildFailed:
```

Comment:
What is this one to fix again?

Reply:
Well this one doesn't usually happen anyone because we started swallowing abnormal closures in the websocket actor. Now it would need to be a websocket actor panic. I prob will just revert this though and clean it up later.